cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -129,20 +132,13 @@ func (o *Options) complete() error {
 	recommendedPluginOrder.Insert(allOrderedPlugins...)
 	o.Recommended.Admission.RecommendedPluginOrder = recommendedPluginOrder.List()
 
+	// TODO: Temporary deletionconfirmation it as disabled by default.
+	// Enable it once the old shoot deletion mechanic is removed.
+	o.Recommended.Admission.DefaultOffPlugins.Insert(deletionconfirmation.PluginName)
 	return nil
 }
 
 func (o *Options) config() (*apiserver.Config, error) {
-	// Enable some admission plugins by default
```

Comment:
why has this been removed?

Reply:
In 1.10 of the `apiserver` library, all plugins in `RecommendedPluginOrder` are enabled by default.

Reply:
Great, thank you!

Reply:
```
--disable-admission-plugins strings     admission plugins that should be disabled although they are in the default enabled plugins list. Comma-delimited list of admission plugins: Initializers, MutatingAdmissionWebhook, NamespaceLifecycle, ValidatingAdmissionWebhook. The order of plugins in this flag does not matter.
--enable-admission-plugins strings      admission plugins that should be enabled in addition to default enabled ones. Comma-delimited list of admission plugins: Initializers, MutatingAdmissionWebhook, NamespaceLifecycle, ValidatingAdmissionWebhook. The order of plugins in this flag does not matter.
```